web/index.php

```diff
@@ -88,9 +82,12 @@
 $app->get('/search', function (Request $request) use ($app, $searchForm) {
     /** @var \Doctrine\DBAL\Query\QueryBuilder $queryBuilder */
     $queryBuilder = $app['db']->createQueryBuilder();
+    $wporgClient  = \Rarst\Guzzle\WporgClient::getClient();
+    $currentPage  = $request->query->get('page', 1);
     $type         = $request->get('type');
     $active       = $request->get('active_only');
```

Comment:
If you are removing the `active_only` option, you should remove this line as well.